### PR TITLE
Fix some mistakes in Polish translation

### PR DESCRIPTION
**Pull request by KrzysiekJ**

### CODE REVIEW

**Comment by ykzts** [comment]
cc @m4sk1n

**Comment by nightpool** [comment]
requires review from one other polish user.

File: app/javascript/mastodon/locales/pl.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
   "emoji_button.search": "Szukaj...",
   "emoji_button.symbols": "Symbole",
   "emoji_button.travel": "Podróże i miejsca",
-  "empty_column.community": "Lokalna oś czasu jest pusta. Napisz coś publicznie, aby odbić piłeczkę!",
+  "empty_column.community": "Lokalna oś czasu jest pusta. Napisz coś publicznie, aby zagaić!",
```
Comment (m4sk1n) [inline app/javascript/mastodon/locales/pl.json]: Maybe more correct, but looks weird… Rest is ok

**Comment by nightpool** [comment]
can be merged with feedback addressed. Thanks for taking a look @m4sk1n!

File: app/javascript/mastodon/locales/pl.json (hunk 1 above)
Comment (KrzysiekJ) [inline app/javascript/mastodon/locales/pl.json]: The previous phrase „odbić piłeczkę” roughly refers to responding in Polish, hence doesn’t look appropriate for a situation in which there is no activity yet.

Comment (m4sk1n) [inline app/javascript/mastodon/locales/pl.json]: Maybe just „aby to zmienić”, or „aby ożywić dyskusję”?

Comment (KrzysiekJ) [inline app/javascript/mastodon/locales/pl.json]: Maybe then „aby rozpocząć dyskusję”? However [according to the PWN dictionary, „zagaić” means roughly the same](https://sjp.pwn.pl/sjp/zagaic;2542265.html) (and is shorter). Either the former or the latter is acceptable for me though.

**Review by mkljczk — APPROVED** [review]

**Review by ykzts — APPROVED** [review]

**Pull request merged**